metaflow/metaflow_environment.py

```diff
@@ -104,11 +103,8 @@ def get_boto3_copy_command(self, s3_path, local_path, command="download_file"):
     def get_package_commands(
             self,
             code_package_url,
-            is_kfp_plugin=False,
     ):
-        mflog_bash_cmd = BASH_MFLOG if not is_kfp_plugin else BASH_MFLOG_KFP
-        cmds = [mflog_bash_cmd,
-                "mflog \'Setting up task environment.\'",
+        cmds = ["mflog \'Setting up task environment.\'",
```

Comment:
comparing to: https://github.com/Netflix/metaflow/blob/master/metaflow/metaflow_environment.py#L81-L84

this is now missing:
```python
        cmds = [
            BASH_MFLOG,
```

Reply:
Good point; unfortunately `BASH_MFLOG` doesn't work for us, so I moved `BASH_MFLOG_KFP` from `kfp.py` into `metaflow_environment.py` and added it in the same way to keep things consistent.

Reply:
Good point, I reverted back to using the `is_kfp_plugin` boolean to substitute in the correct bash command (`BASH_MFLOG` vs `BASH_MFLOG_KFP`).